content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -188,7 +188,7 @@ For a reference to old feature gates that are removed, please refer to
 | `SizeMemoryBackedVolumes` | `false` | Alpha | 1.20 | 1.21 |
 | `SizeMemoryBackedVolumes` | `true` | Beta | 1.22 | |
 | `StatefulSetAutoDeletePVC` | `false` | Alpha | 1.22 | |
-| `StatefulSetStartOrdinal` | `false` | Alpha | 1.26 | |
+| `StatefulSetStartOrdinal` | `true` | Beta | 1.27 | |
```

Comment:
You are not supposed to **remove** the existing line, which is about the duration for the Alpha stage.
You will instead **amend** that line indicating that the Alpha stage stops at `1.26`.
The new line added is correct.

Reply:
Thanks, updated.